core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -2012,7 +2013,8 @@ private[spark] class DAGScheduler(
       execId: String,
       fileLost: Boolean,
       hostToUnregisterOutputs: Option[String],
-      maybeEpoch: Option[Long] = None): Unit = {
+      maybeEpoch: Option[Long] = None,
+      ignoreShuffleVersion: Boolean = false): Unit = {
```

Comment:
Please add this to the java doc. Also I'm not completely sure about the name of the variable.

Reply:
Yeah I will make it be something like `ignoreShuffleFileLostEpoch` to be even more explicit